### ISSUE TITLE
Logging into ignition web

### ISSUE THREAD

**Issue by justinmusgrove** [thirdgen88/ignition-examples#2]
Newbie here w/ ignition/docker. First great work on ignition docker project, really appreciate the examples.

I have followed instructions for [mysql](https://github.com/thirdgen88/ignition-examples/tree/master/ignition-mysql) but when I try to log into the ignition [web UI](http://localhost:8088/web/signin?3) no username/password allow me to log in correctly. 

The [ignition docker](https://github.com/thirdgen88/ignition-docker/tree/master/docs) states 

`$ docker run --name my-ignition \
    -v /path/to/custom/ignition.conf:/var/lib/ignition/data/ignition.conf \
    -e GATEWAY_ADMIN_PASSWORD=password \
    -d kcollins/ignition:tag`

Which I have running successfully and able to login with admin/password

I am probably missing something silly but would appreciate any direction. Thanks. 

*** **Update** ***
I switched over to [postgres](https://github.com/thirdgen88/ignition-examples/tree/master/ignition-postgres) and works like a charm. Username: admin Password: value in secrets file.

**Comment by thirdgen88** [comment]
@justinmusgrove, thanks for the kind words, I'm glad to hear from you!  (also glad to see that the _postgres_ one is working!)  I've attempted a fresh spool-up of the `ignition-mysql` section and was able to log in successfully:

https://asciinema.org/a/303829

That said, I also recently did some updates to the base image to support bringing in additional JDBC drivers from an additional bind-mount.  That wouldn't be needed for logging into the Ignition gateway though, only to successfully complete the database connection from the Gateway after things are spooled up.

Can you attempt to, from within the `ignition-examples/ignition-mysql` folder, clean the slate and re-launch the solution?  (make sure to `docker-compose down` on your Postgres stack beforehand, or you'll get a port conflict on `8088`).

```
cd ignition-examples/ignition-mysql
docker-compose down -v  # Remove all containers AND volumes
docker-compose up -d
```

From there, if you still have trouble logging into the gateway with the `secrets/GATEWAY_PASSWORD`, try retrieving the provisioning log so you can post it here:

```
docker cp ignition-mysql_gateway_1:/usr/local/share/ignition/logs/provisioning.log .
```

If you could post that `provisioning.log` and also the `ignition-mysql.log` that can be generated from `docker logs ignition-mysql_gateway_1 > ignition-mysql.log`, that might help me figure out what went wrong in your setup.

Also, perhaps share your platform that you're working from (Windows, macOS, Linux?  Version of Docker / Docker-Compose)?

Thanks!

**Comment by justinmusgrove** [comment]
Thanks @thirdgen88 for your quick reply. I performed the following steps 

1. reclone repo
2. generated secrets 
3. `docker-compose up -d && docker-compose logs -f`
4. Logged into the ignition UI

And it worked! The only other thing between yesterday and today is I updated my operating system to Mac 10.15.3.

Up until now I was getting around it by setting properties in environment within docker container:
`    environment:
      - GATEWAY_ADMIN_USERNAME=admin
      - GATEWAY_ADMIN_PASSWORD=password
`

Sorry for the false alarm.

Side question - have you thought or seen anyone containerizing the [Ignition Demo Project](https://inductiveautomation.com/downloads/extra-material)?
